src/print/index.ts

```diff
@@ -388,9 +388,12 @@ export function print(path: FastPath, options: ParserOptions, print: PrintFn): D
                 }
 
                 const quotes = !isLoneMustacheTag(node.value) || options.svelteStrictMode;
+                const quoteCharacter = options.svelteMarkupSingleQuote ? '\'' : '"';
+                const quoteReplacementCharacter = options.svelteMarkupSingleQuote ? '"' : '\'';
+                // TODO: Replace quoteCharacter with quoteReplacementCharacter in attrNodeValue
                 const attrNodeValue = printAttributeNodeValue(path, print, quotes, node);
```

Comment:
I need some help here as I don't know how to replace characters in `attrNodeValue`. I'm not yet familiar with the Prettier API.

Reply:
Kindly asking for feedback / help here @Conduitry @dummdidumm @jamesbirtles